ilastik/shell/gui/ilastikShell.py

```diff
@@ -632,14 +635,81 @@ def _loaduifile(self):
+            b = QToolButton(self.startscreen, objectName="NewProjectButton_" + workflow_cls.__name__)
             styleStartScreenButton(b, ilastikIcons.GoNext)
             b.setText(displayName)
-            b.clicked.connect(partial(self.loadWorkflow, workflow))
-            self.startscreen.VL1.insertWidget(pos, b)
-            pos += 1
+            b.clicked.connect(partial(self.loadWorkflow, workflow_cls))
+            return b
+
+        wfs = {
+            wf_name: (wf_class, wf_name, wf_display_name)
+            for wf_class, wf_name, wf_display_name in getAvailableWorkflows()
+        }
+
+        wf_startup_menu_order = {
+            "Segmentation Workflows": {
+                "workflows": [
+                    "Pixel Classification",
+                    "AutocontextTwoStage",
+                    "Neural Network Classification (Local)",
+                    "Neural Network Classification (Remote)",
+                    "Carving",
+                    "Edge Training With Multicut",
+                ],
+                "init_expanded": True,
+            },
+            "Object Classification Workflows": {
+                "workflows": [
+                    "Object Classification (from prediction image)",
+                    "Object Classification (from binary image)",
+                ],
+                "init_expanded": True,
+            },
+            "Tracking Workflows": {
+                "workflows": [
+                    "Manual Tracking Workflow",
+                    "Automatic Tracking Workflow (Conservation Tracking) from binary image",
+                    "Automatic Tracking Workflow (Conservation Tracking) from prediction image",
+                    "Animal Conservation Tracking Workflow from Binary Image",
+                    "Animal Conservation Tracking Workflow from Prediction Image",
+                    "Structured Learning Tracking Workflow from binary image",
+                    "Structured Learning Tracking Workflow from prediction image",
+                ],
+                "init_expanded": False,
+            },
+        }
+
+        # First add workflows according to above groups
+        for group, wf_dict in wf_startup_menu_order.items():
+            group_layout = QVBoxLayout()
+            group_widget = QWidget()
+            group_widget.setLayout(group_layout)
+            for wf_name in wf_dict["workflows"]:
+                try:
+                    wf_class, wf_name, wf_display_name = wfs.pop(wf_name)
+                except KeyError:
+                    warnings.warn(f"Worflow startup menu: Could not find {wf_name}")
+                    continue
+
+                group_layout.addWidget(_tool_button(wf_class, wf_display_name))
+
+            wfs_layout.addWidget(CollapsibleWidget(group_widget, f" {group}", init_expanded=wf_dict["init_expanded"]))
+
+        # add remaining workflows to "Other WFs group"
+        group_layout = QVBoxLayout()
```

Comment:
It's possible to avoid duplication and inline button creation by populating the data table with all workflows that are not explicitly mentioned.

```python
wfs = {
    wf_name: (wf_class, wf_display_name)
    for wf_class, wf_name, wf_display_name in getAvailableWorkflows()
}

explicit_wf_names = {
    wf_name
    for wf_dict in wf_startup_menu_order.values()
    for wf_name in wf_dict["workflows"]
}
wf_startup_menu_order["Other Workflows"] = {
    "workflows": [
        wf_name
        for wf_name in wfs
        if wf_name not in explicit_wf_names
    ],
    "init_expanded": False,
}

for group, wf_dict in wf_startup_menu_order.items():
    buttons = []

    for wf_name in wf_dict["workflows"]:
        try:
            wf_class, wf_display_name = wfs[wf_name]
        except KeyError:
            warnings.warn(f"Worflow startup menu: Could not find {wf_name}")
            continue

        button = QToolButton(
            self.startscreen,
            objectName=f"NewProjectButton_{wf_class.__name__}",
            text=wf_display_name,
            clicked=lambda: self.loadWorkflow(wf_class),
        )
        styleStartScreenButton(button, ilastikIcons.GoNext)
        buttons.append(button)
    
    if not buttons:
        continue
    
    group_layout = QVBoxLayout()
    for button in buttons:
        group_layout.addWidget(button)

    group_widget = QWidget()
    group_widget.setLayout(group_layout)
    
    wfs_layout.addWidget(CollapsibleWidget(group_widget, f" {group}", init_expanded=wf_dict["init_expanded"]))
```